README.md

```diff
@@ -200,9 +204,11 @@ Open the [index.html file](./webapp/static/index.html) and delete the following
 Now add a comment so that Copilot can generate code for you:
 
 ```html
-<!-- create an interactive form using javascript that accepts text as input. The form must display a button and submit an asynchronous request to the /generate API endpoint -->
+<!-- create an interactive a form with text input and button, and add event listener to button to send POST request to /generate endpoint and display response in a div with id "result" -->
 ```
-This should be enough for Copilot to generate code for you after you press `Enter` (or `Return`). The generated code should look like this:
+This should be enough for Copilot to generate code for you after you press `Enter` (or `Return`). 
+If not, use `Ctrl+Enter` to give multiple suggestions, choose one that fits better to the code below.
+The generated code should look like this:
```

Comment:
Can you add a note saying that it is entirely possible that Copilot will not generate the exact same snippet? And in that case, to continue typing or overwriting the generated text to fix it